libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/logger/SalesforceLogger.java

```diff
@@ -71,31 +77,65 @@ public enum Level {
         DEBUG(2),
         VERBOSE(1);
 
-        private Integer severity;
+        private final Integer severity;
```

Comment:
All the inspector warnings in this source were trivial and low risk to resolve, so I included that in the spirt of good housekeeping.  Would it be a good time for a Kotlin conversion on this?